vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1340,6 +1340,15 @@ void LedgerImpl::GetConfirmationsHistory(
       to_timestamp_seconds, callback);
 }
 
+void LedgerImpl::BroadcastReconcile(bool contribution_in_progress) {
```

Comment:
I don't think this method is what we want here. The library doesn't care about "broadcasting" anything. Let's talk in DM about a better way to handle this

Reply:
This has been moved out of native ledger